routes/items.js

```diff
@@ -12,71 +13,70 @@ router.get('/', (req, res) => {
 });
 
 /* GET an item by id */
-router.get('/items/:id', (req, res) => {
+router.get('/:id', (req, res) => {
 	Item.findById(req.params.id)
 		.populate('_user')
 		.then(item => res.status(200).json(response(res, item)));
 });
 
 /* GET an item's images */
-router.get('/item/:id/images', (req, res) => {
+router.get('/:id/images', (req, res) => {
 	Image.find({ '_item': req.params.id }).then(images => res.status(200).json(response(res, images)));
 });
 
 /* POST Create a new item */
-router.post('/items', (req, res) => {
+router.post('/', (req, res) => {
 
 	// validate input fields
+	// this needs to return appropriate error code
 	req.checkBody('name', 'The name is required').notEmpty();
   req.checkBody('quality', 'You must supply the quality of the item').notEmpty();
 
 	const err = req.validationErrors();
 
-	if(err) { return res.status(200).json(response(res, { errors: err })); }
+	if(err) { return res.status(400).json(response(res, err, 'Bad Request')); }
 
 	else {
-
-		const item = new Item({ name: req.body.name, quality: req.body.quality, _user: req.body.user });
-		item.save().then(err => res.status(200).json(response(res, { item: item, err: err })));
+		const newItem = {
```

Comment:
What is the idea of declaring this variable if it is only used once?
I think we should leave it as it was before, it doesn't make sense to spend resources on a single-use variable.